bot/cogs/utils.py

```diff
@@ -87,6 +88,50 @@ def __init__(self, bot: AutoShardedBot):
 
         await ctx.message.channel.send(embed=pep_embed)
 
+    @command()
```

Comment:
Original is correct because of 

```py
from discord.ext.commands import AutoShardedBot, Context, command
```

so it's effectively `@commands.command()`.

Reply:
yeah, that is my point. I don't think so.

Reply:
The parentheses are necessary

Reply:
The decorator accepts options to be passed to Command() first, and returns an inner function that accepts the commmand function. So the parentheses are required:
```py
# normal
@commands.command()
async def my_command(self, ctx):
    ...

# deco as a func
commands.command()(my_command)

# each step shown
get_command_instance_function = commands.command()
command_object = get_command_instance_function(my_command)

# if no parens
get_command_instance_function = commands.command(my_command)
# no command object resulting and original callback passed as arg to wrong function
```